src/app/address/routes/sharedFields.js

```diff
@@ -115,6 +115,107 @@ module.exports = {
       },
     ],
   },
+  nonUKAddressApartmentNumber: {
+    type: "text",
+    validate: [
+      {
+        type: "maxlength",
```

Comment:
I don't see maxLength in the designs, am I missing something?

Reply:
Ok i'll remove